frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -99,7 +100,14 @@ const orgBookOptions: Ref<Array<any>> = ref([]);
 const parcelAccordionIndex: Ref<number | undefined> = ref(undefined);
 const spacialAccordionIndex: Ref<number | undefined> = ref(undefined);
 const validationErrors: Ref<string[]> = ref([]);
-const formUpdated: Ref<boolean> = ref(false);
+const hasFormBeenEdited: Ref<boolean> = ref(false);
+
+const { formUpdated, stopAutoSave } = useAutoSave(async () => {
+  const values = formRef.value?.values;
+  if (values) {
+    await onSaveDraft(values, true);
+  }
+});
```

Comment:
I'm gonna keep the async and await as I have defined the composable callback function with `saveCallback: () => Promise<void>...` and am choosing to keep this so that future uses of the auto save composable allow for async callbacks.